test/screenshots/compare.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eu
+
+WORK_DIR="$(dirname "$0")"
+BIN="$(realpath "$WORK_DIR/../../assets/node_modules/.bin/pixelmatch")"
+
+for img in "$WORK_DIR"/actual/*.png
+do
+	echo "Comparing $img..."
+	"$BIN" "$img" "${img/actual/expected}"
+done
```

Comment:
Another way of doing this would be to upload the screenshots in Argos CI.

Here is how we do this in cozy-ui.

https://github.com/cozy/cozy-ui/blob/0484ad2cd6d38e535a304e27e2d33c61fe690a8e/.travis.yml#L19
https://github.com/cozy/cozy-ui/blob/master/package.json#L20

It is easy then to review and approve the failing screenshots in Argos.